components/script/dom/htmlfontelement.rs

```diff
@@ -77,8 +80,12 @@ impl VirtualMethods for HTMLFontElement {
             &local_name!("face") => AttrValue::from_atomic(value.into()),
             &local_name!("color") => AttrValue::from_legacy_color(value.into()),
             &local_name!("size") => {
-                let length = parse_length(&value);
-                AttrValue::Length(value.into(), length)
+                let size = parse_size(&value);
```

Comment:
Could we have a function to share the code here and above? It seems that a function to parse `DOMString` into `AttrValue` can be split out.